source/features/dim-bots.tsx

```diff
@@ -58,9 +63,6 @@ function init(signal: AbortSignal): void {
 
 	// Undim on mouse focus
 	delegate(dimBots.selector, 'click', undimBots, {signal});
-
-	// Undim on keyboard focus
-	document.documentElement.addEventListener('navigation:keydown', undimBots, {once: true, signal});
```

Comment:
Because GitHub crippled this event, I just have to drop it. There's no way to distinguish between an actual navigation and the mere pressing of the `ctrl` key. So, nothing, I'm not going to rewrite their logic to detect actual j/k navigation.